scripts/newinstall.sh

```diff
@@ -482,17 +486,26 @@ n8l::get_tagged_env() {
     | grep '^#CONDA_ENV=' | cut -d= -f2) \
     || fail "Unable to determine conda env"
   platform="$(uname -s)"
-  case "$platform" in
-    Linux)
+	cpu_arch="$(uname -m)"
+	echo "Platform: $platform-$cpu_arch"
+  if [[ "$platform" = "Linux" ]]; then
+    if [[ "$cpu_arch" = "x86_64" ]]; then
       eups_platform="redhat/el7/conda-system/miniconda3-${LSST_MINICONDA_VERSION}-$env_version"
-      ;;
-    Darwin)
+    elif [[ "$cpu_arch" = "aarch64" ]]; then
+      eups_platform="redhat/el8-arm/conda-system/miniconda3-${LSST_MINICONDA_VERSION}-$env_version"
+    fi
+
+  elif [[ "$platform" = "Darwin" ]]; then
+    if [[ "$cpu_arch" = "x86_64" ]]; then
       eups_platform="osx/10.9/conda-system/miniconda3-${LSST_MINICONDA_VERSION}-$env_version"
-      ;;
-    *)
-      fail "Unknown platform: $platform"
-      ;;
-  esac
+    elif [[ "$cpu_arch" = "arm64" ]]; then
+      eups_platform="osx/14-arm/conda-system/miniconda3-${LSST_MINICONDA_VERSION}-$env_version"
+    fi
+  else
+    fail "Unknown platform or arch: $platform $cpu_arch"
```

Comment:
This won't actually fail on a known platform with unknown arch.  You'd need a check for an empty `eups_platform` to do that.